## Don't use the RTLD_GLOBAL options when calling dlopen(), 
## this should not be needed.  Some broken libraries might fail.
## Patch from upstream CVS.

Index: libtool-1.5.24/libltdl/ltdl.c
===================================================================
--- libtool-1.5.24.orig/libltdl/ltdl.c	2007-07-11 20:24:41.000000000 +0200
+++ libtool-1.5.24/libltdl/ltdl.c	2007-07-11 20:24:59.000000000 +0200
@@ -1068,17 +1068,6 @@
 #  include <sys/dl.h>
 #endif
 
-#ifdef RTLD_GLOBAL
-#  define LT_GLOBAL		RTLD_GLOBAL
-#else
-#  ifdef DL_GLOBAL
-#    define LT_GLOBAL		DL_GLOBAL
-#  endif
-#endif /* !RTLD_GLOBAL */
-#ifndef LT_GLOBAL
-#  define LT_GLOBAL		0
-#endif /* !LT_GLOBAL */
-
 /* We may have to define LT_LAZY_OR_NOW in the command line if we
    find out it does not work in some platform. */
 #ifndef LT_LAZY_OR_NOW
@@ -1114,7 +1103,7 @@
      lt_user_data loader_data;
      const char *filename;
 {
-  lt_module   module   = dlopen (filename, LT_GLOBAL | LT_LAZY_OR_NOW);
+  lt_module   module   = dlopen (filename, LT_LAZY_OR_NOW);
 
   if (!module)
     {
